introspection/README.md

```diff
@@ -36,22 +28,16 @@ Introspection happens in the following steps:
 * The introspection connector takes that db-schema representation and converts it into Prismas internal datamodel representation
 * The internal datamodel representation gets rendered into the Prisma schema
 
-The formats for the connection strings are discussed in the schema spec. https://github.com/prisma/specs/blob/master/schema/Readme.md
+The formats for the connection strings are discussed in the schema spec. https://github.com/prisma/specs/tree/master/schema#supported-fields
 The conversion from the sql-schema to the Prisma internal datamodel changes the information from a description of the data layout / connection in the db to the data structure used to generate the API. This means that some tables / collections that are present in the database are omitted while some fields that have no representation in the database are generated. 
 
 * Backrelation fields for relations that store Ids in the model table
 * Omitting Prisma style M:N relation tables and adding relationfields to their models
 * Omitting Prisma style scalar list tables and instead adding the list fields to their models
 
-The logic Prisma uses to create names for relations / relation fields in cases where they are not specified in the database is also described in the Schema spec. 
+The logic Prisma uses to create names for relations / relation fields in cases where they are not specified in the database is also described in the Schema spec. https://github.com/prisma/specs/tree/master/schema#relations
 
 The renderer then takes the internal datamodel representation and prints it. While doing this, it omits printing out some information if it follows Prismas opionionation. Examples include:
 
 * not printing relation names for unambigous relations if their name matches a Prisma style M:N relation table '_PostToUser'
-* not printing the @relation(references: id) if the foreign key is on the lexicographically lower model
-
-### Current
-
-
-### Future 👽
```

Comment:
The idea about these headlines was to describe the existing JS/TS implementation and then the "new" Rust based one - maybe this is not useful any more.